pkg/doc/signature/suite/jsonwebsignature2020/public_key_verifier_test.go

```diff
@@ -109,8 +90,27 @@ func TestPublicKeyVerifier_Verify_EC(t *testing.T) {
 	})
 }
 
+func mapECCurveToKeyType(curve elliptic.Curve) kmsapi.KeyType {
+	switch curve {
+	case elliptic.P256():
+		return kmsapi.ECDSAP256TypeIEEEP1363
+
+	case elliptic.P384():
+		return kmsapi.ECDSAP384TypeIEEEP1363
+
+	case elliptic.P521():
+		return kmsapi.ECDSAP521TypeIEEEP1363
+
+	case btcec.S256():
+		return kmsapi.ECDSASecp256k1TypeIEEEP1363
+
+	default:
+		panic("unsupported curve")
+	}
+}
```

Comment:
this seems like a utility function that can go in internal as well.

Reply:
I think I can move it to the public `doc/util/signature` package. Or we can add it somewhere at `kms/api` where `kms.KeyType` is defined. What do you think @Baha-sk  ?

Reply:
It should not be moved to `kms/api` because IEEEP1363 format is specific for JWS signing, the KMS offers DER format as well. This function can stay in a utility package of jsonwebsignauture2020 or a parent package.